src/sources/http.rs

```diff
@@ -21,7 +21,7 @@ impl Source for HTTPSource {
     fn get_ip<'a>(&'a self, family: Family) -> IpFuture<'a> {
         async fn run(_self: &HTTPSource, family: Family) -> IpResult {
             trace!("Contacting {:?}", _self.url);
-            let client = reqwest::Client::builder();
+            let client = reqwest::Client::builder().timeout(std::time::Duration::from_secs(30));
```

Comment:
it would be better to add a field to the HTTPSource struct with the timeout and use 30 seconds as the default